dss-api.yml

```diff
@@ -32,17 +32,97 @@ paths:
       parameters:
         - name: uuid
           in: path
-          description: File unique ID.
+          description: A RFC4122-compliant ID for the file.
           required: true
           type: string
         - name: replica
           in: query
           description: Replica to fetch from.
-          required: true
+          required: false
+          type: string
+        - name: timestamp
+          in: query
+          description: Timestamp of file creation in RFC3339.  If this is not provided, the latest version is returned.
+          required: false
           type: string
+          format: date-time
       responses:
+        200:
+          description: On a HEAD request, returns metadata
+          headers:
+            # edits to here should probably be reflected in the 302 section immediately below.
+            X-DSS-BUNDLE_UUID:
```

Comment:
Explanation: I used dashes for content-type because it's actually dash'ed in common usage.  I'm using _ as a space replacement.

Happy to change it to dashes everywhere if you think that's better.